### PR TITLE
feat: Adds missing CSS modules

### PR DESCRIPTION
**Pull request by lucasfp13**
## What's the purpose of this pull request?

This PR intends to clean as many imports from `src/styles/globals/components` as possible. Additionally, we simplified `CartToggle` badge to be a regular badge for a `Button` with only an icon as its content.

We added CSS modules on:
- BannerText
- ScrollToTopButton
- SrOnly
- Badge

Also, this PR contains some minor changes and code refactoring.

## How to test it?

All components should look as it was before.

## Checklist
**PR Title and Commit Messages**
- [x] PR title and commit messages follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) specification
  - Available prefixes: `feat`, `fix`, `chore`, `docs`, `style`, `refactor`, `perf`, and `test`

**PR Description**
- [x] Added a label according to the PR goal - `Breaking change`, `Features`, `Bug fixes`, `Chore`, `Documentation`, `Style changes`, `Refactoring`, `Performance`, and `Test`
- [x] Added the component, hook, or path name in-between backticks (\`\`) - *if applicable, e.g., `ComponentName` component, `useWindowDimensions` hook*

**Documentation**
- [x] PR description
- [x] Added to/Updated the Storybook - *if applicable*

### CODE REVIEW

File: src/components/common/Navbar/navbar.module.scss [hunk 1]
```diff
@@ -98,10 +107,6 @@
     align-items: center;
     justify-content: flex-end;
 
-    [data-fs-button-collapse] {
-      margin-left: calc(-1 * var(--fs-spacing-3));
```
Comment (hellofanny) [inline src/components/common/Navbar/navbar.module.scss:102]: cool! the left margin seems fixed 👍 

|Before|After|
|-|-|
image|image

File: src/components/regionalization/RegionalizationBar/RegionalizationBar.stories.mdx [hunk 2]
```diff
@@ -20,9 +20,7 @@ export const TemplateRegionalizationModal = () => {
 export const TemplateBarWithProvider = () => {
   return (
     <UIProvider>
-      <div
-        style={{ display: 'flex', alignItems: 'center', columnGap: '20px' }}
-      >
+      <div style={{ display: 'flex', alignItems: 'center', columnGap: '20px' }}>
```
Comment (hellofanny) [inline src/components/regionalization/RegionalizationBar/RegionalizationBar.stories.mdx:23]: I think we can also update the component path here (L50)

`src/components/regionalization/RegionalizationBar`

image

File: src/components/sections/ScrollToTopButton/scroll-to-top-button.module.scss [hunk 3]
```diff
@@ -1,6 +1,6 @@
 @import "src/styles/scaffold";
 
-.scroll-to-top-button {
+.fs-scroll-to-top-button {
```
Comment (hellofanny) [inline src/components/sections/ScrollToTopButton/scroll-to-top-button.module.scss:3]: Do u think we can rename it to `ScrollTopButton`?  🤔

**Review by hellofanny — APPROVED** [review]
I've tested those changes on the [storybook preview](https://nextjs-store-storybook-git-feat-adds-missing-c-6e6f85-faststore.vercel.app/?path=/docs/atoms-button--overview) and [page preview](https://sfj-2f90ad1--nextjs.preview.vtex.app/). Everything seems to work fine 👍

File: src/components/sections/ScrollToTopButton/scroll-to-top-button.module.scss (hunk 3 above)
Comment (renatamottam) [reply src/components/sections/ScrollToTopButton/scroll-to-top-button.module.scss:3]: I like that!

**Review by renatamottam — APPROVED** [review]
LGTM!

**Pull request merged**